src/fs.h

```diff
@@ -19,6 +21,21 @@ namespace fs = boost::filesystem;
 namespace fsbridge {
     FILE *fopen(const fs::path& p, const char *mode);
     FILE *freopen(const fs::path& p, const char *mode, FILE *stream);
+
+    class FileLock
+    {
+    public:
+        FileLock(const fs::path& file);
```

Comment:
nit: could delete default and copy constructor

Reply:
Fixed